src/Main.java

```diff
@@ -50,45 +50,53 @@ public static void main(String[] args) {
 
         // Задача 5
 
-        int ageOfTheChild = 15;
+        int ageOfTheChild = 20;
 
         if (ageOfTheChild < 5) {
             System.out.println(" Если возраст ребенка равен " + ageOfTheChild + ", то ему нельзя кататься на аттракционе.");
         }
 
-        if (ageOfTheChild > 5 && ageOfTheChild < 14 ) {
-            System.out.println(" Если возраст ребенка равен " + ageOfTheChild + ", то он может кататься только в сопровождении взрослого. Если взрослого нет, то кататься нельзя.");
+        else if (ageOfTheChild >= 5 && ageOfTheChild  < 14 ) {
+            System.out.println(" Если возраст ребенка равен " + ageOfTheChild + ", то он может кататься только в сопровождении взрослого. Если взрослого нет, то кататься нельзя." );
+
         } else {
-            System.out.println(" Если возраст ребенка равен " + ageOfTheChild + ", то он может кататься без сопровождения взрослого.");
+            System.out.println( "Если возраст ребенка равен " + ageOfTheChild + ", то ему можно кататься на аттракционе без сопровождения взрослого.");
         }
 
         // Задача 6
 
-        int aSeatInTheCarriage = 65;
 
-        if (aSeatInTheCarriage > 102) {
+        int wagonCapacity = 102;
+        int seatingCapacity = 60;
+        int numberOfOccupiedSeats = 50;
+        int numberOfOccupiedStandingPlaces = 10 ;
+
+        int Person = numberOfOccupiedSeats + numberOfOccupiedStandingPlaces;
```

Comment:
```suggestion
        int person = numberOfOccupiedSeats + numberOfOccupiedStandingPlaces;
```
названия переменных пишем с маленькой буквы всегда